pidstat: add page (#3529)
tldr-pages/tldr: pages/linux/pidstat.md

# pidstat

> Show system resource usage, including CPU, memory, IO etc.

> - Show CPU statistics at two second intervals for 10 times:

Author: hi @pxgamer @schneiderl  please take a look at this description. I am not sure how to put the placeholder into the description.

Reviewer: What about this?
```suggestion
- Show CPU statistics at a given interval a given number of times:
```

> `pidstat {{interval}} {{times}}`

Reviewer 2: Let us use concrete values for "interval" and "times", and update the description to match those values.

- Show page faults and memory utilization:

`pidstat -r`

> - Show io usage per process id:

Reviewer 3: Perhaps using the long form of this would be a bit clearer?
```suggestion
- Show input/output usage per process id:
```

`pidstat -d`

> - Show specific PID:

Reviewer 2: Show information on a specific PID

`pidstat -p {{PID}}`

> - Show memory statistics for all processes whose command name inclues string "fox" or "bird":

Reviewer 2: Let us remove "string"

> `pidstat -C "fox|bird" -r -p ALL`

Reviewer 3: Just missing the token syntax here:
```suggestion
`pidstat -C "{{fox|bird}}" -r -p ALL`
```
